### PR TITLE
Carbon black rewrite

### PR DESCRIPTION
**Pull request by MosheEichler**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added in the readme file some differences between v1 & v2  of the carbon black endpoint standard integration

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandard/README.md [hunk 1]
```diff
@@ -3,7 +3,23 @@ This integration was integrated and tested with version 1.1.2 of Carbon Black En
...
+* [cbd-find-events-details](#cbd-find-events-details) Retrieve details for enriched events.
+
+3 new playbooks
+1. "Find Events" Finds events using a search query (or device_id, etc.).
+2. "Find Event Details" Receives event IDs and returns details about the event.
+3. "Find Processes" Find processes using a search query (or device_id, etc.).
+
+Also added:
+* Mapper.
+* Layout.
```
Comment (reutshal) [inline Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandard/README.md:22]:
```suggestion
An incident type, classifier and mapper were added.
```

File: Packs/CarbonBlackDefense/ReleaseNotes/2_0_1.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
 #### Integrations
-##### Carbon Black Endpoint Standard
+##### Carbon Black Endpoint Standard v2
```
Comment (reutshal) [inline Packs/CarbonBlackDefense/ReleaseNotes/2_0_1.md:3]: Revert this change, no need to change old RN

File: Packs/CarbonBlackDefense/ReleaseNotes/2_0_2.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Carbon Black Endpoint Standard v2
+- Added the differences between the two versions in the README file.
```
Comment (reutshal) [inline Packs/CarbonBlackDefense/ReleaseNotes/2_0_2.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

**Review by reutshal — CHANGES_REQUESTED** [review]

**Comment by reutshal** [comment]
@ShirleyDenkberg Can you please review the changes added to the README file? 
Thanks 🙏

File: Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandard/README.md [hunk 4]
```diff
@@ -3,7 +3,23 @@ This integration was integrated and tested with version 1.1.2 of Carbon Black En
 
 ## The changes v1
 The old integration is deprecated because Carbon Black have released a new version of their API.
-The new integration supports new commands, Also added a Mapper and a Layout.
+The new integration supports new commands:
```
Comment (yaakovi) [inline Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandard/README.md:6]: is there a difference in the old commands? did we change inputs/outputs?
did we stop supporting specific commands (if so, why?)

**Comment by ShirleyDenkberg** [comment]
@yaakovi @reutshal I edited the README file as we discussed.

**Review by reutshal — APPROVED** [review]

**Pull request merged**